### PR TITLE
Itsi neap params fix

### PR DESCRIPTION
**Pull request by masha1997**

### CODE REVIEW

File: provider/resource_notable_event_aggregation_policy.go [hunk 1]
```diff
@@ -779,10 +807,6 @@ func notableEventAggregationPolicy(ctx context.Context, d *schema.ResourceData,
 		rule_json["description"] = _rule["description"].(string)
 		rule_json["priority"] = _rule["priority"].(int)
 
-		rule_json["activation_criteria"], err = criteria("activation_criteria", _rule["activation_criteria"].(*schema.Set))
```
Comment (masha1997) [inline provider/resource_notable_event_aggregation_policy.go:782]: moved to 790 line

File: provider/resource_notable_event_aggregation_policy.go [hunk 2]
```diff
@@ -1007,7 +1026,11 @@ func populateNotableEventAggregationPolicyResourceData(ctx context.Context, b *m
 	default:
 		diag.FromErr(fmt.Errorf("usupported type %s for priority", t))
 	}
-	err = d.Set("split_by_field", strings.Split(interfaceMap["split_by_field"].(string), ","))
+
+	if split_by_field, ok := interfaceMap["split_by_field"]; ok && split_by_field != "" {
```
Comment (masha1997) [inline provider/resource_notable_event_aggregation_policy.go:1030]: drift on empty split_by_field fix

File: provider/resource_notable_event_aggregation_policy.go [hunk 3]
```diff
@@ -1215,51 +1238,49 @@ func populateNotableEventAggregationPolicyResourceData(ctx context.Context, b *m
 						if name, ok := config.(map[string]interface{})["name"]; ok {
 							_name := name.(string)
 							if params, ok := config.(map[string]interface{})["params"]; ok {
-								for _, supported_action := range supported_execute_actions {
```
Comment (masha1997) [inline provider/resource_notable_event_aggregation_policy.go:1218]: just replace array with supported_execute_actions map

**Review by benoittoulme — DISMISSED** [review]
supported_execute_actions is a map 👍

File: provider/resource_notable_event_aggregation_policy.go [hunk 4]
```diff
@@ -634,7 +654,7 @@ func ResourceNotableEventAggregationPolicy() *schema.Resource {
 func criteria(criteria_type string, criteria_data *schema.Set) (criteria map[string]interface{}, err error) {
 	criterias := criteria_data.List()
```
Comment (dmitrykruglov) [inline provider/resource_notable_event_aggregation_policy.go:655]: not a big deal, but the word `criteria` is already plural, the singular noun is `criterion` :)

File: provider/resource_notable_event_aggregation_policy.go [hunk 5]
```diff
@@ -1215,51 +1238,49 @@ func populateNotableEventAggregationPolicyResourceData(ctx context.Context, b *m
...
-												matches := rex.FindAllStringSubmatch(value.(string), -1)
-												extra_params := map[string]interface{}{}
-												for _, v := range matches {
-													extra_params[v[1]] = v[2]
+									tf_execute_action := map[string]interface{}{}
+
+									// trim action,%%action_name%% prefix
+									for key, value := range execute_action {
+										trimmed_key := strings.TrimPrefix(key, "action."+_name+".")
+										// apllicable for slack_adv, bigpanda_stateful
```
Comment (dmitrykruglov) [inline provider/resource_notable_event_aggregation_policy.go:1258]:
```suggestion
										// applicable to slack_adv, bigpanda_stateful
```

**Review by dmitrykruglov — DISMISSED** [review]

**Review by dmitrykruglov — APPROVED** [review]

**Pull request merged**